pkg/config/dumper.go

```diff
@@ -45,6 +49,7 @@ type DumpConfig struct {
 	S3BucketName             string       `json:"s3_bucket_name"`
 	DynamoTableName          string       `json:"dynamo_table_name"`
 	PostgresConnectionString string       `json:"postgres_connection_string"`
+	ThirdRailContext         bool         `json:"third_rail_context"`
```

Comment:
This is where things get really messy. It's only going to attempt to fix names using aliases if we're in "third-rail" mode, indicating that all the other tables are present. We also set the schema slightly differently, since we can't have foreign keys for tables that don't exist.